src/components/ModalBase/ModalBase.jsx

```diff
@@ -0,0 +1,51 @@
+import { Box, Button, IconButton, Modal, Typography } from "@mui/material";
+import { useState } from "react";
+import { modalStyle, titleStyles } from "./Modal.styles";
+import ClearIcon from "@mui/icons-material/Clear";
+
+const ModalBase = ({
+  titleModal,
+  descriptionModal,
+  renderBody,
+  renderFooter,
+}) => {
+  const [open, setOpen] = useState(false);
+  const handleOpen = () => setOpen(true);
+  const handleClose = () => setOpen(false);
+  return (
+    <div>
+      <Button onClick={handleOpen}>Open modal</Button>
```

Comment:
you should use the "New Delivery" button to open the modal

Reply:
Ok 👍